### ISSUE TITLE
Copying swagger default.yaml file from the config folder

### ISSUE THREAD

**Issue by ghost** [apigee-127/swagger-sails#3]
i think we need to copy the default.yaml file from the config folder to the sails config folder while using swagger as a hook in sails

**Comment by theganyo** [comment]
That is correct. The default.yaml should be in the sails config directory. If you create a project using the "swagger" CLI, it should do this for you. Are you seeing a issue with this?
